ci/prow/config_knative.yaml

```diff
@@ -35,27 +35,20 @@ presubmits:
           memory: 16Gi
     - unit-tests: true
     - integration-tests: true
-      args:
-      - "--run-test"
-      - "./test/e2e-tests.sh --istio-version 1.2-latest --no-mesh"
+      full-command: "./test/e2e-tests.sh --istio-version 1.2-latest --no-mesh"
```

Comment:
This will work, but you're loosing all the presubmit checks by supplying the full command.

I suggest rolling back the previous PR if the jobs are broken, then properly passing the flags. I think this is unsupported by presubmit-tests.sh, so it will need a small update.